<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_normal">
       <attribute name="title">
        <string>常规模式</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="2,1,10">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_calculate">
          <property name="text">
           <string>计算</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
           <item row="0" column="0">
            <widget class="QLabel" name="label_expression">
             <property name="text">
              <string>表达式：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_expression"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_result">
             <property name="text">
              <string>结果：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_result">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_tip">
          <property name="text">
           <string>双击列表项选择常量或者参数</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1,1,2">
           <item row="1" column="2">
            <widget class="QListWidget" name="listWidget_constants">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>常量列表</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="pushButton_add_function">
             <property name="text">
              <string>添加函数</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>说明</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>函数列表</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_add_constant">
             <property name="text">
              <string>添加常量</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="pushButton_modify">
             <property name="text">
              <string>修改或删除</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QListWidget" name="listWidget_functions">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_complex">
       <attribute name="title">
        <string>复数模式</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_big">
       <attribute name="title">
        <string>大数计算</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
